Frends.AzureEventHub.Receive/Frends.AzureEventHub.Receive/AzureEventHub.cs

```diff
@@ -35,7 +36,7 @@ public static async Task<Result> Receive([PropertyTab] Consumer consumer, [Prope
         if (options.MaxRunTime > 0 && consumer.MaximumWaitTime > options.MaxRunTime)
             throw new Exception("Consumer.MaximumWaitTime cannot exceed Options.MaxRunTime when Options.MaxRunTime is greater than 0.");
 
-        var results = new List<dynamic>();
+        var results = new ConcurrentBag<dynamic>();
         var errors = new List<dynamic>();
```

Comment:
I think that errors are also used in a concurrent context, so those would need the concurrentbad as well.